tfhe/src/core_crypto/experimental/commons/parameters.rs

```diff
@@ -0,0 +1,41 @@
+use serde::{Deserialize, Serialize};
+
+/// The number of elements in a partial GLWE secret key that are drawn from the random distribution.
+#[derive(Debug, PartialEq, Eq, Copy, Clone, Serialize, Deserialize)]
+pub struct PartialGlweSecretKeyRandomCoefCount(pub usize);
```

Comment:
Is this used anywhere?

Reply:
not in this PR, it's related to partial GLWE secret key generation which I put in a separate commit

Reply:
will put in the appropriate commit/PR